README.md

```diff
@@ -378,6 +378,17 @@ Sometimes we need to use third-party component in our forms. But don't you worry
 
 Below are some examples with [react-select](https://github.com/JedWatson/react-select) and [react-datepicker](https://github.com/Hacker0x01/react-datepicker/).
 
+_Observation_: If your are using Eslint + Prettier you need to take care with your `useEffect` dependecies, probably they will change when you format your code. To fix that you can add a comment on the line above of your `useEffect` dependencies disabling the eslint on that line. You can just follow the example below.
+
+_Example:_
+
+```js
+useEffect(() => {
+  registerField();
+  // eslint-disable-next-line
+}, [ref.current, fieldName]);
```

Comment:
Personally I feel like we should add the comment on the current line, not on the one above it. This is to reduce the chance of someone missing it (or adding something in between).

```js
useEffect(() => {
  registerField();
}, [ref.current, fieldName]); // eslint-disable-line
```

Also, I think it would be nice to add the `// eslint-disable-line` on our current examples, so when they copy it, the ESLint rule won't apply to the `useEffect` dependencies line.

Reply:
Sorry I don't understand what you are trying to explain, would you like to add this comment on the examples about that already are on the documentation or replace the `// eslint-disable-next-line` whit a the  `// eslint-disable-line`?

Reply:
Fixed